stdlib/ssl.pyi

```diff
@@ -344,17 +414,18 @@ class SSLContext:
         session: Optional[SSLSession] = ...,
     ) -> SSLObject: ...
     def session_stats(self) -> Dict[str, int]: ...
-    if sys.version_info >= (3, 7):
-        hostname_checks_common_name: bool
-        maximum_version: TLSVersion
-        minimum_version: TLSVersion
 
 class SSLObject:
-    context: SSLContext
-    server_side: bool
-    server_hostname: Optional[str]
-    session: Optional[SSLSession]
-    session_reused: bool
+    @property
+    def context(self) -> SSLContext: ...
```

Comment:
These all also have a setter, so the old code is probably more correct (a property with a getter and a setter is nearly equivalent to what we had). Not sure if any type checkers would care about the difference between a get/set property and a bare instance variable.

Reply:
Looks like mypy-primer found the same thing applies to SSLContext.

Reply:
ok, will change it, I thought it is better to reflect the original code, but if mypy-primer complains about it 😄